root/release/edit/tracklist.tt

```diff
@@ -206,18 +206,18 @@
     </td>
 
     <td class="position">
-      <input class="pos" type="text" data-bind="value: number" />
+      <input class="pos" type="text" data-bind="value: number, event: { keydown: $root.onTrackInputKeyDown }" />
```

Comment:
We have a plugin called `knockout-delegatedEvents` which extends knockout to allow adding a single delegated event handler on a parent element, rather than installing separate handlers on every track. Which in theory may be more efficient for releases with many tracks.

To use that, it should be as simple as changing the parent medium table to:

```HTML
<table class="medium tbl" data-bind="delegatedHandler: 'keydown'">
```

and then replacing the `event` binding handlers with a `data-keydown` attribute:

```HTML
<input class="pos" type="text" data-bind="value: number" data-keydown="onTrackInputKeyDown" />
```

(We probably should have done something similar with the `focusTrackName` handler. IIRC the handler should still receive the specific context (track) as an argument, but maybe not as the first argument.)

Reply:
Seems to work. Thanks for the suggestion!